.markdownlint.json

```diff
@@ -1,3 +0,0 @@
-{
-    "line-length": { "line_length": 140 }
```

Comment:
eslint:recommended ruleset does not contain by default the [line-lenght](https://eslint.org/docs/rules/max-len) setting. Add this setting to the eslint or this file

Reply:
There is a default line-length setting, which is 80 characters. Disscussed this with @hexagon06 and decided to stick to the default. This deleted file was actually ignored by eslint, so all lines were at most 80 characters already.